src/rvary_fmt_plug.c

```diff
@@ -56,7 +56,6 @@ static struct fmt_tests rvary_tests[] = {
 static char (*saved_key)[PLAINTEXT_LENGTH + 1];
 static uint32_t (*crypt_out)[BINARY_SIZE / sizeof(uint32_t)];
 static DES_key_schedule (*schedules);
-static int dirty;
```

Comment:
Great, but at the same time `schedules` become unneeded, to be replaced with a variable local to the loop.